### PR TITLE
Feature/optimize dsl code

### PR DESCRIPTION
**Pull request by nidomiro**

### CODE REVIEW

File: kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/SchemaBuilder.kt [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
-import com.apurebase.kgraphql.schema.model.TypeDef
-import com.apurebase.kgraphql.schema.model.MutableSchemaDefinition
-import com.apurebase.kgraphql.schema.structure.SchemaCompilation
 import kotlin.reflect.KClass
 
 /**
  * SchemaBuilder exposes rich DSL to setup GraphQL schema
  */
-class SchemaBuilder<Context : Any>(private val init: SchemaBuilder<Context>.() -> Unit) {
+class SchemaBuilder {
```
Comment (jeggy) [inline kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/SchemaBuilder.kt]: We could make it `class SchemaBuilder internal constructor() {` to make sure people are using `KGraphQL.schema {}`

File: kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/SubscriptionDSL.kt [hunk 2]
```diff
@@ -1,54 +1,34 @@
-package com.apurebase.kgraphql.schema.dsl
-
-import com.apurebase.kgraphql.Context
-import com.apurebase.kgraphql.schema.*
-import com.apurebase.kgraphql.schema.model.*
+package com.apurebase.kgraphql.schema.dsl.operations
+
+import AbstractOperationDSL
```
Comment (jeggy) [inline kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/SubscriptionDSL.kt]: This doesn't look right

Comment (nidomiro) [reply kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/SubscriptionDSL.kt]: What do you mean by that?

File: kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/SchemaBuilder.kt (hunk 1 above)
Comment (nidomiro) [reply kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/SchemaBuilder.kt]: done ;)

File: kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/SubscriptionDSL.kt (hunk 2 above)
Comment (jeggy) [reply kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/SubscriptionDSL.kt]: found out what is wrong. This line should be removed and you are missing package definition in `AbstractOperationDSL`

Comment (nidomiro) [reply kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/dsl/operations/SubscriptionDSL.kt]: You are right. I have overseen that, thanks. (it's fixed)

**Pull request merged**